Formula/gcc.rb

```diff
@@ -1,7 +1,7 @@
 class Gcc < Formula
   desc "GNU compiler collection"
   homepage "https://gcc.gnu.org/"
-  if Hardware::CPU.arm?
+  if OS.mac? && Hardware::CPU.arm?
```

Comment:
Does this build for you on ARM Linux with this change?

Reply:
Yup, it does.